SportHub/src/modules/AdminMainArticlesSection/components/AdminMainArticlesSection.js

```diff
@@ -0,0 +1,117 @@
+import React, {useEffect, useState} from 'react'
```

Comment:
GENERAL RECOMMENDATION: Do you have any ideas how to make this component simpler / move some logic in children components